client/app/start.go

```diff
@@ -286,7 +252,7 @@ func makeBaseAppOpts(cfg Config) ([]func(*baseapp.BaseApp), error) {
 	}
 
 	return []func(*baseapp.BaseApp){
-		// baseapp.SetOptimisticExecution(), // TODO: Enable this.
```

Comment:
This is an experimental feature and may have impact on things that we don't know. Let's not enable it for now.

Reply:
Instead, let's create an issue to track this

Reply:
Sure, I got it confused with optimistic building. Disabled in 0498250